lib/active_model_serializers/serializable_resource.rb

```diff
@@ -2,7 +2,7 @@
 
 module ActiveModelSerializers
   class SerializableResource
-    ADAPTER_OPTION_KEYS = Set.new([:include, :fields, :adapter, :meta, :meta_key, :links])
+    ADAPTER_OPTION_KEYS = Set.new([:include, :fields, :adapter, :meta, :meta_key, :links, :serialization_context])
```

Comment:
@remear this is where I added serialization context to the adapter options.  

Now pagination links call from jsonapi adapter is `PaginationLinks.new(serializer.object, instance_options).as_json` and PaginationLinks calls `@context = adapter_options.fetch(:serialization_context)`